src/components/logo/Logo.jsx

```diff
@@ -0,0 +1,8 @@
+import logo from "./3.png";
```

Comment:
The image should be moved to `/src/assets` and it should have an obvious name like for example `logo` or `logo__image`.